libsolidity/ast/AST.h

```diff
@@ -1216,7 +1216,7 @@ class EventDefinition: public CallableDeclaration, public StructurallyDocumented
 	FunctionTypePointer functionType(bool /*_internal*/) const override;
 
 	bool isVisibleInDerivedContracts() const override { return true; }
-	bool isVisibleViaContractTypeAccess() const override { return false; /* TODO */ }
+	bool isVisibleViaContractTypeAccess() const override { return true; /* TODO */ }
```

Comment:
Please remove the todo when you are confident that your change is correct and is not a breaking change.

Reply:
Also add this to the changelog if it is a change.